docs/release-notes/0.9/release-0.9.md

```diff
@@ -0,0 +1,48 @@
+# ML.NET 0.9 Release Notes
+
+Welcome to 2019! For the past 9 months we have been working on adding features and improving [ML.NET](https://aka.ms/mlnet). In the remaining releases before we reach 1.0, we are focusing more on the overall stability of the package. To acheive that we will continue to work on refining the API, increasing test coverage and improving documentation. 0.9 release packs multiple fixes as well as significant clean up to the internal code of the package.
```

Comment:
Saying we're at 0.9, then mentioning 1.0, I think the normal reader will assume 1.0 comes right after 0.9.

Also, `acheive ` should be `achieve`.

```suggestion
Welcome to 2019! For the past 9 months we have been working on adding features and improving [ML.NET](https://aka.ms/mlnet). In the remaining releases before we reach 1.0, we are focusing more on the overall stability of the package. To achieve that we will continue to work on refining the API, in the forthcoming 0.10, 0.11, and 0.12 releases, to increase the test coverage and improving the documentation. The 0.9 release packs multiple fixes as well as significant clean up to the internal code of the package.
```